dotcom-rendering/src/web/layouts/DecideLayout.tsx

```diff
@@ -15,9 +16,41 @@ type Props = {
 	article: FEArticleType;
 	NAV: NavType;
 	format: ArticleFormat;
+	renderingTarget: RenderingTarget;
 };
 
-export const DecideLayout = ({ article, NAV, format }: Props) => {
+const decideLayoutApps = ({
```

Comment:
I found it difficult to express this! Most likely there's a better way!

Reply:
I think this makes sense, for improvement I'd suggest

1. Renaming to `DecideLayoutApps`
2. Breaking out the existing switch statement into an (in file) `DecideLayoutWeb` component
3. Having the DecideLayout component simply pick between either `DecideLayoutApps` or `DecideLayoutWeb`